leetcode/twoSum.js

```diff
@@ -0,0 +1,16 @@
+function twoSum(nums, target) {
+  let targetSum;
```

Comment:
avoid `let` whenever possible, the less you use `let` the easier it is to read the code.

just do
```
const targetSum = nums[i] + nums[j];
```